.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: Test
```

Comment:
Find inspiration on the syntax here: 
https://github.com/nextcloud/app-tutorial/blob/master/.github/workflows/node.yml

Reply:
I duplicated our lint.yml

seems that needs an update as well, so let's do it afterwards.

Reply:
Yeah, it's just so we have similar layout in the tests :)
Easier for us to manage and to set the branch protections (otherwise each repo have different test names :p